### DISCUSSION TITLE
npm/ci badges in markdown are not rendered correctly

### DISCUSSION THREAD

**Discussion by sedghi** [remarkjs/remark#937] [Q&A] [answered]
I'm using github graphql to fetch Readme files from github and use `ReactMarkdown` to render them as html in our website. 
As you know github readmes often have badges like 

![image](https://user-images.githubusercontent.com/7490180/149852351-c1a0b439-080b-4fde-9358-eff84da7caf8.png)

which is the result of the following section in the markdown

```
[![NPM version][npm-version-image]][npm-url]
[![NPM downloads][npm-downloads-image]][npm-url]
[![Pulls][docker-pulls-img]][docker-image-url]
[![MIT License][license-image]][license-url]
[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2FOHIF%2FViewers.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2FOHIF%2FViewers?ref=badge_shield)

[![Netlify Status][netlify-image]][netlify-url]
[![CircleCI][circleci-image]][circleci-url]
[![codecov][codecov-image]][codecov-url]
[![This project is using Percy.io for visual regression testing.][percy-image]](percy-url)
[![All Contributors](https://img.shields.io/badge/all_contributors-10-orange.svg?style=flat-square)](#contributors)
```

`ReactMarkdown` renders them as 

![image](https://user-images.githubusercontent.com/7490180/149852395-6a8f7d84-1009-46f5-b532-3d93e0ec26ba.png)

Any recommendation how I can manage to render them the same way? (two rows with gaps between them)

**Comment by ChristianMurphy** [comment, marked as answer]
@sedghi could you share a runnable example of the issue?
I tried the structure you shared (replacing some of the images not linked) and it produces two rows as expected
https://codesandbox.io/s/react-markdown-badges-tzm42

It likely isn't `react-markdown` itself.
It could be related to:
* enabling https://github.com/remarkjs/remark-breaks
* custom css applied to the page

**Reply by sedghi** [reply]
Thanks for your prompt reply, yes, it was a css issue, my `img` tag had display block on it
